testing/conf.lua

```diff
@@ -8,3 +9,14 @@ function love.conf(t)
   t.window.stencil = true
   t.renderers = {"opengl"}
 end
+
+-- custom crash message here to catch anything that might occur with modules 
+-- loading before we hit main.lua
+local function error_printer(msg, layer)
+	print((debug.traceback("Error: " .. tostring(msg), 1+(layer or 1)):gsub("\n[^\n]+$", "")))
+end
+function love.errorhandler(msg)
+	msg = tostring(msg)
+	error_printer(msg, 2)
+  return nil
```

Comment:
indentation (this line can just be removed too, since not having a `return` statement does the same thing)